rollup.config.js

```diff
@@ -33,7 +33,7 @@ export default [
     external: ['react', 'react-dom'],
   },
   {
-    input: 'dist/esm/index.d.ts',
+    input: 'dist/esm/types/index.d.ts',
```

Comment:
How would you test this @lkappeler? I linked this package to `seller-web` locally and it sounds good, is that enough?

Reply:
Missed your comment...
You can check if the types are there in the target file and if your IDE is showing you the typings of the imported functions and objects properly.

Reply:
Thank you 👍 It looks fine then!